### PR TITLE
Streamtube

### PR DESCRIPTION
**Pull request by emmanuelle**
Doc upgrade checklist:

- [x] random seed is set if using random data
- [x] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [x] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [x] `graph_objs` has been renamed to `graph_objects`
- [x] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [x] `fig.show()` at the end of each example
- [x] `px` example at the top if appropriate
- [x] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

### CODE REVIEW

**Comment by emmanuelle** [comment]
Note that further improvements of the streamtube plot should probably wait until https://github.com/plotly/plotly.js/issues/4206 is solved.

File: python/streamtube-plot.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+fig.add_trace(go.Streamtube(x=x, y=y, z=z, u=u, v=w, w=v), 1, 3)
+
+fig.update_layout(scene_camera_eye=dict(x=2, y=2, z=2),
+                  scene2_camera_eye=dict(x=2, y=2, z=2),
+                  scene3_camera_eye=dict(x=2, y=2, z=2))
+fig.show()
+```
+
+#### Reference
+See https://plot.ly/python/reference/ for more information and chart attribute options!
```
Comment (nicolaskruchten) [inline python/streamtube-plot.md]: can you link directly to the trace docs plz? i.e. add the `#streamtube` at the end?

Comment (emmanuelle) [reply python/streamtube-plot.md]: oh yes sure

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**